dartagnan/src/main/java/com/dat3m/dartagnan/parsers/program/visitors/VisitorLitmusC.java

```diff
@@ -318,14 +331,32 @@ public Expression visitReC11SCmpXchg(LitmusCParser.ReC11SCmpXchgContext ctx) {
     }
 
     @Override
-    public Expression visitReC11WCmpXchg(LitmusCParser.ReC11WCmpXchgContext ctx) {
+    public Expression visitReC11SCmpXchg(LitmusCParser.ReC11SCmpXchgContext ctx) {
```

Comment:
Are you sure you are implementing the correct one here? `W` is for weak CAS; `S` for strong CAS

Reply:
Yeah the weak CAS ones are at line 343-358